ceph-defaults/defaults/main.yml

```
@@ -13,6 +13,10 @@ ceph_pkgs:
   - cephadm
   - podman
   - ceph-common
+ceph_client_pkgs:
+  - chrony
+  - ceph-common
+client_conf_dir: "{{ playbook_dir}}/.config-{{ client_group}}"
```

```suggestion
client_conf_dir: "{{ playbook_dir }}/.config-{{ client_group }}"
```
beyond that, this might be unnecessary, see the suggested approach below.

---

Marking as resolved - latest push contains your suggested slurp approach.